hivemind/server/layers/__init__.py

```diff
@@ -10,7 +11,12 @@
 
 name_to_input = {'ffn': lambda batch_size, hid_dim: torch.empty((batch_size, hid_dim)),
                  'transformer': lambda batch_size, hid_dim:
-                 (torch.empty((batch_size, 128, hid_dim)), torch.empty((batch_size, hid_dim), dtype=torch.bool)),
+                 (torch.empty((batch_size, 128, hid_dim)), torch.empty((batch_size, 128), dtype=torch.bool)),
                  'nop': lambda batch_size, hid_dim: torch.empty((batch_size, hid_dim)),
                  'det_dropout': lambda batch_size, hid_dim:
                  (torch.empty((batch_size, hid_dim)), torch.randint(0, 1, (batch_size, hid_dim)))}
+
+expert_name_to_scheduler = {'ffn': get_linear_schedule_with_warmup, 'transformer': get_linear_schedule_with_warmup,
```

Comment:
arguably the default should be none - that way we do not run a risk of accidentally using a schedule (and the config will always contain the schedule explicitly.

In contrast, if we keep this as a default, we will have to break compatibility whenever we decide upon a new better schedule

Reply:
Replaced and removed expert_name_to_scheduler